src/validation.cpp

```diff
@@ -2808,8 +2808,13 @@ bool CChainState::ConnectBlock(const CBlock& block, CValidationState& state, CBl
             auto storedGovVars = cache.GetStoredVariables(static_cast<uint32_t>(pindex->nHeight));
             for (const auto& var : storedGovVars) {
                 CCustomCSView govCache(cache);
-                // Ignore any Gov variables that fail to validate, apply or be set.
-                if (var->Validate(govCache) && var->Apply(govCache, pindex->nHeight) && govCache.SetVariable(*var)) {
+                // Add to existing ATTRIBUTES instead of overwriting.
+                if (var->GetName() == "ATTRIBUTES") {
+                    auto govVar = mnview.GetVariable(var->GetName());
```

Comment:
Same as above, check for nullptr

Reply:
Will do. I do not think that should happen but let's add check to make sure.